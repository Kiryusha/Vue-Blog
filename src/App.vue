<template lang="pug">
  .page
    #app
      Header
      router-view
</template>

<script>
import Header from '@/components/layout/header'

export default {
  name: 'App',
  components: {
    Header
  }
}
</script>

<style lang="stylus">
  .page
    min-height 100vh
    font-family 'Open Sans', sans-serif
    color #3
    padding-bottom 250px

  .container
    max-width 980px
    margin auto

    +mobile()
      padding 0 15px

</style>
