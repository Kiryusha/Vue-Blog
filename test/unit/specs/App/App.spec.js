// App.vue

import { shallowMount } from '@vue/test-utils';

import App from '@/App';

describe('App.vue', () => {
  describe('component shallow rendering', () => {
    it('should match a snapshot', () => {
      const wrapper = shallowMount(App, { stubs: ['router-view'] });

      expect(wrapper.html()).toMatchSnapshot();
    });
  });
});
