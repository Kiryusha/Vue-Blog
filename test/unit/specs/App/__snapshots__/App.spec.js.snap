// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.vue component shallow rendering should match a snapshot 1`] = `
<div class="page">
  <div id="app">
    <auth-stub></auth-stub>
    <response-stub></response-stub>
    <header-stub></header-stub>
    <transition-stub name="fade"></transition-stub>
  </div>
</div>
`;
